### PR TITLE
[ERSSUP-56070]-[]-[Add _ping to CallHeathCheck]-[DMW]

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by nhsd-hubert-raich — APPROVED** [review]

**Review by nhsd-hubert-raich — APPROVED** [review]

**Comment by nhsd-hubert-raich** [comment]
All good by me, but you will have to rebase and rerun the pipelines as one seem to have failed with intermittent issue.

**Review by nhsd-jack-wainwright — CHANGES_REQUESTED** [review]
I think there should also be some changes around retrieving the status monitoring API key from the API-M secure variables.

File: proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml [hunk 1]
```diff
@@ -1,8 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Get any eRS specific Key Value Map parameters -->
```
Comment (nhsd-jack-wainwright) [inline proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml:2]: As discussed over slack, given we're not currently using a Key Value Map in Apigee we can probably remove this policy for now.

**Review by nhsd-jack-wainwright — APPROVED** [review]
Looks good to me 🙂

**Review by nhsd-hubert-raich — APPROVED** [review]

**Review by nhsd-jack-wainwright — APPROVED** [review]
Looks good to me 🙂

**Pull request merged**